src/controller/python/chip-device-ctrl.py

```diff
@@ -855,14 +855,15 @@ def do_getfabricid(self, line):
                 print("Unexpected argument: " + args[1])
                 return
 
-            fabricid = self.devCtrl.GetFabricId()
+            fabricid = self.devCtrl.GetCompressedFabricId()
         except exceptions.ChipStackException as ex:
             print("An exception occurred during reading FabricID:")
             print(str(ex))
             return
 
         print("Get fabric ID complete")
-        print("Fabric ID: " + hex(fabricid))
+        print("Compressed Fabric ID: " +
+              hex(fabricid) + " (" + str(fabricid) + ")")
```

Comment:
Suggest using `%08X` or similar, to ensure format is always 0-padded 64-bit hex value. Suggest also including the raw fabric ID, so that when root public key is more cleanly handled, all items can be provided: <root pk, fabric id> and compressed fabric ID